### DISCUSSION TITLE
Which takes priority when both a LIB view and a SPEF view are loaded for a certain module?

### DISCUSSION THREAD

**Discussion by donn** [The-OpenROAD-Project/OpenROAD#3271] [Q&A]
Assuming we:

```tcl
read_liberty x.lib
read_spef x.spef
```

Which one does OpenROAD/OpenSTA consider when performing static timing analysis and is it dependent on load order?

**Comment by maliberty** [comment]
Lib cells are black boxes so the spef is ignored afaik.  I don't think the spef would annotate in this case.
